src/lib.rs

```diff
@@ -441,6 +441,22 @@ impl Rng {
         }
     }
 
+    /// Fill a byte slice with random data.
+    #[inline]
+    pub fn fill(&self, slice: &mut [u8]) {
+        for item in slice {
+            *item = self.u8(..);
+        }
+    }
+
+    /// Return a byte array of the given size.
+    #[inline]
+    pub fn bytes<const N: usize>(&self) -> [u8; N] {
```

Comment:
Won't this be an MSRV bump?

Reply:
Hmm, if it is, we'll have to decide whether it's worth it or not I guess.

Reply:
I've removed this function for now, can open another PR fot it.